dashboard/lib/widgets/lattice.dart

```diff
@@ -906,3 +908,15 @@ FlutterErrorDetails _debugReportException(FlutterErrorDetails details) {
   FlutterError.reportError(details);
   return details;
 }
+
+/// A [MaterialScrollBehavior] that supports mouse dragging.
+class _MouseDragScrollBehavior extends MaterialScrollBehavior {
+  static _MouseDragScrollBehavior _instance;
+  static _MouseDragScrollBehavior get instance => _instance ??= _MouseDragScrollBehavior();
+
+  @override
+  Set<PointerDeviceKind> get dragDevices => <PointerDeviceKind>{
+    PointerDeviceKind.touch,
+    PointerDeviceKind.mouse,
```

Comment:
Does this support the scroll wheel? A common operation I did was shift+scroll wheel to go horizontally

Reply:
That doesn't work for me. But then, I think Aaron pointed out that shift + scroll wheel works for him now in production, and I can't reproduce that either, so that may be a mac thing (I'm on Linux)?